### PR TITLE
PP-449 updating README.md

### PR DESCRIPTION
**Pull request by PauloPortugal**
## WHAT

_A brief description of the pull request:_
- Adding the email notification endpoints documentation
- Removing the gateway account information on a GET
- Changing JSON keys from camelCase to snake_case
## WHO

_Who should review this pull request:_
- [X] Developers
- [X] WebOps

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -442,6 +444,83 @@ Content-Type: application/json
 |  `events[0].updated`        | X | The date and time of the event                      |
 
 
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+### GET /v1/api/accounts/{accountId}/email-notification
+
+This endpoint retrieves Updates an email notification template body for account `accountId`
```
Comment (imranbohoran) [inline README.md]: The statement says "retrieves Updates an email..". Should it instead just be "retrives an email..."

File: README.md [hunk 2]
```diff
@@ -442,6 +444,83 @@ Content-Type: application/json
...
+HTTP/1.1 404 Not Found
+Content-Type: application/json
+Content-Length: 72
+
+{
+    "message": "The gateway account id '111' does not exist"
+}
+```
+
+#### Response if mandatory fields are missing
```
Comment (imranbohoran) [inline README.md]: Assuming that the mandatory field is "custom-email-text". Are there any other validations worth mentioning here that might come up with 400? I don't think so, but just checking anyways

Comment (PauloPortugal) [inline README.md]: No. Just that one.

File: README.md (hunk 1 above)
Comment (PauloPortugal) [inline README.md]: Thanks! Fixed.

**Comment by PauloPortugal** [comment]
rebuild

**Comment by PauloPortugal** [comment]
rebuild

File: README.md [hunk 3]
```diff
@@ -442,6 +444,83 @@ Content-Type: application/json
 |  `events[0].updated`        | X | The date and time of the event                      |
 
 
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+### GET /v1/api/accounts/{accountId}/email-notification
+
+This endpoint updates an email notification template body for account `accountId`
```
Comment (imranbohoran) [inline README.md]: Shouldn't this be a retrieves an email notification?

Comment (rahul-vinayak) [inline README.md]: I think this endpoint name is a bit confusing, as I understand the story covers updating the templates for the emails and enabling sending emails. Should the end points be specific to that?

Comment (PauloPortugal) [inline README.md]: @rahul-vinayak I will revisit them in another PR.

**Comment by bazbremner** [comment]
rebuild

**Comment by PauloPortugal** [comment]
rebuild

**Comment by PauloPortugal** [comment]
rebuild

**Comment by imranbohoran** [comment]
👍

**Pull request merged**